x11-libs/wxGTK/wxGTK-2.8.12.1-r2.ebuild

```diff
@@ -0,0 +1,189 @@
+# Copyright 1999-2015 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Header: /var/cvsroot/gentoo-x86/x11-libs/wxGTK/wxGTK-2.8.12.1-r1.ebuild,v 1.13 2015/03/31 20:05:20 ulm Exp $
+
+EAPI="5"
+WANT_AUTOCONF=2.5
```

Comment:
It will be a build-time breakage, so better than forcing dep that people will forget to update :).